app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class SessionsController < ApplicationController
+  #ISNECUD
```

Comment:
Oh okay, gotcha. For what it's worth, the order doesn't matter to make them functional. I'd probably go for `index`, `show`, `new`, `create`, `edit`, `update`, `destroy`